docs/config/config.xml

```diff
@@ -60,7 +60,12 @@
                 <seq type="numeric" length="4" min='1000' max='9999'/>
             </structure>
         </ExternalID>
-
+        <!--Header table is diplayed on top of instrument list page and instrument pages -->
+        <HeaderTable>
+            <!--Specify the table as the tag and the field to query as the value inside the tag -->
+            <candidate>ProbandDoB</candidate>
+            <parameter_session>MRI_Done</parameter_session>
```

Comment:
that is: shouldn't parameter_candidate also have an example, but github ate my comment because of the angle brackets that I used originally..